src/vs/languages/sass/common/parser/sassParser.ts

```diff
@@ -53,6 +53,11 @@ export class SassParser extends cssParser.Parser {
 
 	// Sass variables: $font-size: 12px;
 	public _parseVariableDeclaration(panic:scanner.TokenType[]=[]): nodes.VariableDeclaration {
+		var cssVariableDeclaration= super._parseCssVariableDeclaration(panic);
```

Comment:
@aeschli This is a good catch. Yes, it is not correct. It is correctly done in Less parser where css variables are parsed only during fall back to rule set parsing. Did a similar change to Sass parser.